src/components/Heading/Heading.module.css

```diff
@@ -5,81 +5,64 @@
 \*------------------------------------*/
 
 /**
- * 1) h1, h2, h3, h4, h5, h6
- * 2) Styles are either controlled by as or size declaration
+ * Remove top and bottom margins applied by browser for headings.
  */
 .heading {
-  @mixin reset;
```

Comment:
I wonder if we were getting anything else from the reset mixin besides removing margin.

Reply:
It also removes padding, which never existed on heading